ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/README.md

```diff
@@ -45,6 +45,7 @@ The following properties are written to JCR for this Form Text component and are
 12. `./maximumMessage` - the message showed to the user if the entered value is more than the maximum value
 13. `./leadDigits` - the max no of digits before decimal that can be entered if the type of field is Decimal.
 14. `./fracDigits` - the max no of digits after decimal that can be entered if the type of field is Decimal.
+15. `./displayFormat` - define the regex pattern complied with the unicode pattern (Can be found [here](https://unicode.org/reports/tr35/tr35-numbers.html#Number_Format_Patterns)).
```

Comment:
unicode patterns are not regex, instead, it typically refers to a template or a set of symbols that define how dates and times should be formatted.